packages/fxa-content-server/app/tests/spec/views/sign_up.js

```diff
@@ -1141,9 +1141,8 @@ describe('views/sign_up', function() {
       // wait for tooltip
       return p.delay(50).then(() => {
         assert.equal($('.tooltip-suggest').text(), 'Did you mean gmail.com?✕');
-        // there are exactly 3 elements with tabindex in the page (show
-        // password button has not been added to the page).
-        assert.equal($('[tabindex]').length, 3);
```

Comment:
fragile tests like this make me 🤬

i'd attempt making it better but i don't even know what it's actually testing